metron-stellar/stellar-common/README.md

```diff
@@ -411,6 +412,14 @@ In the core language functions, we support basic functional programming primitiv
     * format - string
     * arguments... - object(s)
   * Returns: A formatted string.
+  
+### `FUZZY_SCORE`
+  * Description: Returns the Fuzzy Score which indicates the similarity score between two strings. One point is given for every matched character. Subsequent matches yield two bonus points. A higher score indicates a higher similarity.
+  * Input:
+    * string - The full term that should be matched against.
+    * string - The query that will be matched against a term.
+    * string - The IETF BCP 47 language code to use.
```

Comment:
Right, just to the doc string that gets printed when I run `?FUZZY_SCORE` in the REPL.